packages/api/tests/endpoints/granules/helpers.js

```diff
@@ -17,34 +14,37 @@ const { getBucketsConfigKey } = require('@cumulus/common/stack');
 
 const { getDistributionBucketMapKey } = require('@cumulus/distribution-utils');
 
-const {
-  fakeFileFactory,
-  fakeGranuleFactoryV2,
-} = require('../../../lib/testUtils');
+const getPgFilesFromGranuleCumulusId = async (knex, filePgModel, postgresGranuleCumulusId) =>
+  await filePgModel.search(knex, {
+    granule_cumulus_id: postgresGranuleCumulusId,
+  });
```

Comment:
Refactored this a bit to remove `file_name` from the query. This will just get all files for a granule. There's more opportunity for something to go wrong if we have a test not cleaning up after itself or this is used in `serial` perhaps but it does save some complexity.

Reply:
Hmm. 

Yeah, fair.  👍🏻    I think that's fine given we should be using unique databases for separate tests.